modules/uncertainty/R/generate_joint_ensemble_design.R

```diff
@@ -51,29 +52,34 @@ generate_joint_ensemble_design <- function(settings,
     sampled_inputs[[input_tag]] <- input_result$ids
     design_list[[input_tag]] <- input_result$ids
   }
-
-  # Sample parameters
-  PEcAn.uncertainty::get.parameter.samples(
-    settings,
-    ensemble.size = ensemble_size,
-    posterior.files,
-    ens.sample.method
-  )
-
+  
+  # Sample parameters if we don't have it.
+  if (is.null(ensemble_samples)) {
+    PEcAn.uncertainty::get.parameter.samples(
+      settings,
+      ensemble.size = ensemble_size,
+      posterior.files,
+      ens.sample.method
+    )
+    samples.file <- file.path(settings$outdir, "samples.Rdata")
+  }
```

Comment:
Add `if-control` for the case where we don't assign the `ensemble_samples` argument.